pkg/yang/modules.go

```diff
@@ -442,3 +442,9 @@ func (ms *Modules) include(m *Module) error {
 	}
 	return nil
 }
+
+// ClearEntryCache clears the entryCache containing previously converted nodes
+// used by the ToEntry function.
+func (ms *Modules) ClearEntryCache() {
+	ms.entryCache = map[Node]*Entry{}
```

Comment:
Suggest to clear both caches, since `Process` can't be called multiple times anyways.

A fix that allows multiple calls to `Process` should tolerate both caches being cleared between calls.

Reply:
What's the other cache (besides `ms.entryCache` you are suggesting to clear?)

Reply:
`mergedSubmodule`, but actually on another look, I'm concerned `ToEntry()` might not work anymore if that is also cleared. So scratch what I said, this PR looks good as-is.